index.js

```diff
@@ -48,18 +62,23 @@
 
 // console.log(sameSameButDifferent(123, "122")) 
 // // => "different"
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+console.log("%cQuestion 5", "color: red")
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+
+function updateGrade(student, grade) {
```

Comment:
in Ruby, how would we update a value on a hash? Maybe something like...

`student[:grade] = 12` 

How about in Javascript? [Check out property accessors for one or two ways to do this!](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors)